types/cards.ts

```diff
@@ -254,7 +254,7 @@ export type BusinessVirtualCreditCard = BaseCard & {
 
 export type CreateDebitCardRequest = CreateIndividualDebitCardRequest | CreateBusinessDebitCardRequest | CreateIndividualVirtualDebitCardRequest | CreateBusinessVirtualDebitCardRequest
 export type CreateCreditCardRequest = CreateBusinessCreditCardRequest | CreateBusinessVirtualCreditCardRequest
-export type CreateCardRquest = CreateDebitCardRequest | CreateCreditCardRequest
+export type CreateCardRequest = CreateDebitCardRequest | CreateCreditCardRequest
```

Comment:
I would avoid changing this - it's not a backwards compatible change and would most likely be difficult to upstream to Unit

Reply:
I included this fix to keep the changes here: https://github.com/movesfinancial/unit-node-sdk/pull/23 that appear to be reverted in `fork` branch.